package.json

```
@@ -26,8 +26,9 @@
     "camelcase": "^6.2.1",
     "case-sensitive-paths-webpack-plugin": "^2.4.0",
     "ccsm-cds-with-tests": "^0.4.11",
```

```suggestion
    "ccsm-cds-with-tests": "^0.4.12",
```
Should we bump `ccsm-cds-with-tests` to 0.4.12, which adds the Episode of Care queries in our CQL? See https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/119/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519.

---

I'm going to add a commit after running `npm install`, so that the version is updated in package-lock.json as well.